### DISCUSSION TITLE
<span style="color: #b296a6; font-size: 90px; font-weight: 666; text-decoration: underline; text-decoration: line-through; text-decoration-thickness: 5px">来沈阳指定没你好果汁吃</span>

### DISCUSSION THREAD

**Discussion by afatduckcr** [share121/inter-knot#240] [其他]
<span style="color: red">全体目光向我看齐昂  看我看我  我宣布个事儿  我是个傻逼！</span>
![123](https://github.com/user-attachments/assets/7aac16db-a772-4f7f-a4c0-e901af3d05bb)

**Comment by NOVAQUQ** [comment]
牛逼啊大哥

**Comment by wHenry-002** [comment]
我勒个文艺复兴

**Comment by Nahida1205** [comment]
听到了！很有精神！

**Comment by nUMa-alkaid** [comment]
![Uploading 1722049124841.jpg…]()

**Comment by FWzGeek** [comment]
这个委托我接了

**Comment by wugan-bot156** [comment]
好好好

**Comment by kano0x0** [comment]
狼哥怎么是染上以太侵蚀症了吗，好吓人

**Comment by afatduckcr** [comment]
刚看到有人直接复制我这的标题改参数就发出去了，可是我这写的有点问题，有两个text-decoration，当时没注意到...
总之如果你也想发这样的标题，记得删一个"text-decoration: xxx;"，想要下划线就留下text-decoration: underline;，要删除线就留下text-decoration: line-through;，另外，color是颜色，font-size是字体大小，font-weight是字体粗细，text-decoration-thickness是下划线或删除线的粗细。
最好不要把font-size改得特别大，万一影响环境被作者修复就没得玩了。
(好像只能在标题使用，正文用不了)

**Comment by KUNKUN4** [comment]
顶
